tests/Doctrine/Migrations/Tests/Stub/AbstractMigrationWithoutDownStub.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Migrations\Tests\Stub;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+class AbstractMigrationWithoutDownStub extends AbstractMigration
+{
+    public function up(Schema $schema) : void
+    {
+    }
+}
```

Comment:
I think that kind of stub is seldom reused. Have you considered using an anonymous class for this?

Reply:
yes, that was my first attempt, but was hard to test this https://github.com/doctrine/migrations/pull/952/files#diff-e75b76ba40ceb8b96f42333c2ca4dcdcR33 as the class name was variable.

Reply:
Something like `No down() migration implemented for "class@anonymous/home/goetas/projects/doctrine-migrations/tests/Doctrine/Migrations/Tests/AbstractMigrationTest.php0x7f7a5aafa468"`